src/js/components/Profile/ProfileUserCard.module.css

```diff
@@ -28,11 +37,20 @@
     border-radius: var(--xl-border-radius);
 }
 
+.profileImgMe:hover {
+    cursor: pointer;
+    -webkit-filter:brightness(60%);
+    -moz-filter:brightness(60%);
+    filter: url(#brightness); /* required for FF */
+    filter:brightness(60%);
```

Comment:
A za pomocą opacity próbowałeś?

Reply:
Za pomocą opacity nie przejdzie, ponieważ zdjęcie stałoby się przezroczyste i widać by było to drugie. To rozwiązanie, które wykorzystałem jest wadliwe?